lib/postcode_validation/domain/address_list.rb

```diff
@@ -0,0 +1,21 @@
+module PostcodeValidation
+  module Domain
+    class AddressList
+      def initialize(address_row)
+        @address_row = address_row
+      end
+
+      def street_address
+        address_row.fetch('StreetAddress')
```

Comment:
Definitely some dangers here having the Domain object know about the PCA field names - it means your use case unit tests will need to be aware of the PCA field names (opening the unit test up to OCP violation)